aiutils/tftools/batch_normalizer.py

```diff
@@ -0,0 +1,94 @@
+"""A helper class for managing batch normalization state.
+
+This class is designed to simplify adding batch normalization
+(http://arxiv.org/pdf/1502.03167v3.pdf) to your model by
+managing the state variables associated with it.
+
+Important use note:  The function get_assigner() returns
+an op that must be executed to save the updated state.
+A suggested way to do this is to make execution of the
+model optimizer force it, e.g., by:
+
+  update_assignments = tf.group(bn1.get_assigner(),
```

Comment:
Do you think there's a mistake in this example here? The line `optimizer = tf.group(update_assignments)` doesn't make much sense to me, especially considering this is the second grouping of the updates. Also, I assume we'd just run 
```python 
# skipping quite a bit
loss = create_loss_graph()
train = optimizer.minimize(loss)
with sess.as_default():
     # get the update assignments
     ...
     update_assignments.run()
     train.run(batch_feed_stuff)
```

I know the documentation for the moving average also calls tf.group twice for some reason. Am I missing something?